### ISSUE TITLE
Scaling to multi node training

### ISSUE THREAD

**Issue by fminkin** [NVIDIA/NeMo#407]
I have some questions regrading learning rate scaling in QuartzNet / Jasper paper.

Do you scale learning rate as a linear function of batch size? Do you use LARC?
In my experiments, linear scaling is not speeding up the training, while sqrt one is.

**Comment by borisgin** [comment]
We used [NovoGrad](https://arxiv.org/abs/1905.11286). 
increase LR as sqrt (B/B0) and icrease weight decay

**Comment by fminkin** [comment]
Thanks for the insight! 

How much do I need to increase wd? Can you share some empirical results?
